CONTRIBUTING.md

```diff
@@ -20,17 +20,17 @@ don't hesitate to leave a message on the [community forum](https://discourse.alg
 - [Commit conventions 🤓](#commit-conventions-)
 - [Branches organization 🌳](#branches-organization-)
 - [Requirements](#requirements)
-- [Launch the dev environment 👩‍💻](#launch-the-dev-environment-%E2%80%8D)
+- [Launch the dev environment 👩‍💻](#launch-the-dev-environment-)
 - [Updating the examples](#updating-the-examples)
-- [Folders of the project 🗺](#folders-of-the-project-%F0%9F%97%BA)
+- [Folders of the project 🗺](#folders-of-the-project-)
   - [The source folder](#the-source-folder)
   - [The documentation source](#the-documentation-source)
   - [The dev app](#the-dev-app)
 - [Tests 🔬](#tests-)
   - [Unit tests](#unit-tests)
   - [Functional tests](#functional-tests)
 - [Linting ✨](#linting-)
-- [Release 🛳](#release-%F0%9F%9B%B3)
+- [Release 🛳](#release-)
```

Comment:
🤔 why did you change these links? When I try it out, the old ones work and your change doesn't 🙂 

Thanks!

Reply:
Some emojis don't work in links, this one included.